leaf-ui/rappid-extensions/BackboneModels.js

```diff
@@ -1 +1,27 @@
-/** This file contains backbone model representations of the original model objects - WIP */
\ No newline at end of file
+/** This file contains backbone model representations of the original model objects - WIP */
+
+/** 
+ * Backbone for UserEvaluation 
+ * */ 
+var UserEvaluationModel = Backbone.Model.extend({  
+    //Take in properties
+    initialize: function(options){ //Named arguments and passes in parameter
+        this.intentionID = options.intentionID; 
+        this.absTime = options.absTime;
+        this.evaluationValue = options.evaluationValue;
```

Comment:
Change `this.evaluationValue` to be `this.assignedValue

Reply:
>Also, can we change this.absoluteValue to this.absTP.
>And this.absTime to this.absTP.


So will both this.absTime and this.absoluteValue become this.absTP